### ISSUE TITLE
aria-expanded not used for Accordion and Toggle widgets

### ISSUE THREAD

**Issue by besweeet** [elementor/elementor#11246]
## Prerequisites
- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
To better support web accessibility, aria-expanded should be used in both the Accordion and Toggle widgets. This would support the following WCAG criteria:

> 4.1 Compatible - 4.1.2 Name, Role, Value (A) - Ensure that Assistive Technologies can gather information about, activate and keep up to date on the status of user interface controls in the content.

**Comment by besweeet** [comment]
Bump... The most recent versions of Elementor and Elementor Pro do not resolve this important accessibility issue.

**Comment by besweeet** [comment]
Another bump...

**Comment by besweeet** [comment]
Bump... This is probably something that can take developers a couple minutes to resolve.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.1.3

Feel free to update

Thanks!

**Comment by Bovelett** [comment]
YESSSS!! @shilo-ey Thank you! ❤️
